File: Wells/Source/Graphics/PremadeSeriesInfo.vb
Public Class PremadeSeriesInfo
    Property ParameterGroup As String
    Property PropertyDisplayName As String
    Property IsFromWell As String

    Public Overrides Function ToString() As String
        If IsFromWell Then
            Return PropertyDisplayName
        End If
        Return "Precipitaciones"
    End Function
End Class

File: Wells/src/Graphics/PremadeSeriesInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Wells.View.Graphics
{
    public class PremadeSeriesInfo
    {
        public string ParameterGroup { get; set; }
        public string PropertyDisplayName { get; set; }
        public bool IsFromWell { get; set; }

        public override string ToString()
        {
            if (IsFromWell) return PropertyDisplayName;
            return "Precipitaciones";
        }
      
    }
}
